packages/module/patternfly-docs/content/extensions/component-groups/examples/MultiContentCard/MultiContentCardExample.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import MultiContentCard from "@patternfly/react-component-groups/dist/dynamic/MultiContentCard";
+import { Button, Card, CardHeader, CardBody, Text, TextContent, TextVariants, Icon, TextList, TextListItem } from '@patternfly/react-core';
+import { ArrowRightIcon, BellIcon, CogIcon, LockIcon } from '@patternfly/react-icons';
+
+const cards = [
+  <Card isFullHeight isPlain key="card-1">
+    <CardHeader>
+      <TextContent>
+        <Text component={TextVariants.h4}>Getting Started</Text>
+      </TextContent>
+    </CardHeader>
+    <CardBody className="pf-v5-u-display-flex">
+      <TextContent className="pf-v5-u-display-flex pf-v5-u-flex-direction-column pf-v5-u-align-self-stretch">
+        <Text
+          component={TextVariants.p}
+          className="pf-v5-u-font-size-sm pf-v5-u-font-weight-bold pf-v5-u-mb-sm pf-v5-u-link-color-hover"
+        >
+          <Icon size="md" className="pf-v5-u-pl-sm pf-v5-u-pr-md">
+            <CogIcon />
+          </Icon>
+          Configure applications
+        </Text>
+        <Text
+          component="p"
+          className="pf-u-flex-grow-1 pf-v5-u-font-size-sm"
+        >
+          To prepare for integration with the Hybrid Cloud Console,
+          you must configure incoming webhooks in your third-party
+          applications.
+        </Text>
+        <TextList className="pf-u-ml-0 pf-v5-u-font-size-sm">
+          <TextListItem>
+            <Button variant="link" isInline>
+              Configure Slack
+            </Button>
+          </TextListItem>
+          <TextListItem>
+            <Button variant="link" isInline>
+              Configure Google Chat
+            </Button>
+          </TextListItem>
+          <TextListItem>
+            <Button variant="link" isInline>
+              Configure Microsoft Teams
+            </Button>
+          </TextListItem>
+        </TextList>
+      </TextContent>
+    </CardBody>
+  </Card>,
+  <Card isFullHeight isPlain key="card-2">
+    <CardBody className="pf-v5-u-display-flex pf-v5-u-pt-3xl-on-md">
+      <TextContent className="pf-v5-u-display-flex pf-v5-u-flex-direction-column pf-v5-u-align-self-stretch">
```

Comment:
I think all theses utility classes can be replaced by a Flex layout with modifiers for `direction={{ default: 'column' }}` and use the various spacing features :https://www.patternfly.org/layouts/flex#spacing to match this demo: https://www.patternfly.org/patterns/dashboard/html-demos/basic/